components/global/product_switcher/product_switcher.tsx

```diff
@@ -70,11 +84,25 @@ const MenuItemTextContainer = styled.div`
     line-height: 20px;
 `;
 
-const LinkIcon = styled.i`
-    width: 14px;
-    height: 14px;
-    color: rgba(var(--center-channel-color-rgb), 0.56);
-`;
+const SwitcherNavEntry = (props: SwitcherNavEntryProps) => {
+    return (
+        <MenuItem
+            to={props.destination}
+        >
+
+            <IconButton icon={props.icon}/>
```

Comment:
props.icon is a component not a string. This will break the current integrations. Can we put this back to be more flexible?

Reply:
I am not sure on this point since we will replace all existing icons with the compass one anyway which takes a string as the icon glyph. But this shouldn't be an icon button for sure, i will replace it.

Reply:
@nevyangelova This will break the current implementation that Boards has already merged resulting in a white screen on community. Probably easier to just set this back to `{props.icon}`? Or you can work with them to make sure their implementation is changed.